extensions/jupiter-aggregator/src/compare-token-prices.tsx

```diff
@@ -0,0 +1,67 @@
+import { Clipboard, Action, ActionPanel, Form, showToast, Toast } from "@raycast/api";
+import { fetchTokenList, compareTokens } from "./api";
+import { useState } from "react";
+
+export default function Command() {
+  const { data, isLoading } = fetchTokenList();
+  const [token1, setToken1] = useState("SOL");
+  const [token2, setToken2] = useState("USDC");
+  const [amount, setAmount] = useState("1");
+
+  const submitHander = async () => {
+    const price = await compareTokens(token1, amount, token2);
+    await showToast({
```

Comment:
I found that showing a Toast on success was a bit confusing when I was testing this. For a second, I thought the extension didnt work, because the UI didnt change. 

I think we need to improve the UX expectation here. Im thinking that there could be two:
1. Show the value
2. Copy the value

If 1, we chould update the view to a `Detail`, where we show the value in markdown, with a `Action` that allows users to copy it if they want

If 2, we should use a (`showHUD`) upon copying, as it automatically closes Raycast, and we should ensure the primary action is "Calculate and Copy", or something that manages users expectation better

Thanks!

Reply:
Good thoughts here! I'll work on implementing these

Reply:
Added!